<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-weight:600;&quot;&gt;Reading Tip App&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Choose the type</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QComboBox" name="typeComboBox">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>Book</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Podcast</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Video</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Blog post</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QStackedWidget" name="input_form">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="book_page">
           <layout class="QFormLayout" name="formLayout_2">
            <item row="5" column="0">
             <widget class="QLabel" name="commentLabel_2">
              <property name="text">
               <string>Comment</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLineEdit" name="commentLineEdit_2"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="linkLabel">
              <property name="text">
               <string>Tags</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="linkLineEdit"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="iSBNLabel">
              <property name="text">
               <string>ISBN</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="titleLineEdit"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="titleLabel">
              <property name="text">
               <string>Title</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="iSBNLineEdit"/>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="nameLineEdit"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="nameLabel">
              <property name="text">
               <string>Author</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="podcast_page">
           <layout class="QFormLayout" name="formLayout_4">
            <item row="0" column="0">
             <widget class="QLabel" name="nameLabel_2">
              <property name="text">
               <string>Author</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="nameLineEdit_2"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="titleLabel_2">
              <property name="text">
               <string>Title</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="iSBNLineEdit_2"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="iSBNLabel_2">
              <property name="text">
               <string>Description</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="titleLineEdit_2"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="tagsLabel_2">
              <property name="text">
               <string>Tags</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="tagsLineEdit_2"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="commentLabel_4">
              <property name="text">
               <string>Comment</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="commentLineEdit_4"/>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>URL</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLineEdit" name="lineEdit"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="video_page">
           <layout class="QFormLayout" name="formLayout_5">
            <item row="1" column="0">
             <widget class="QLabel" name="titleLabel_3">
              <property name="text">
               <string>Title</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="iSBNLineEdit_3"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="iSBNLabel_3">
              <property name="text">
               <string>URL</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="titleLineEdit_3"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="tagsLabel">
              <property name="text">
               <string>Tags</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="tagsLineEdit"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="commentLabel">
              <property name="text">
               <string>Comment</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="commentLineEdit"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="blogpost_page">
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="titleLabel_4">
              <property name="text">
               <string>Title</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="iSBNLineEdit_4"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="nameLabel_4">
              <property name="text">
               <string>Author</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="titleLineEdit_4"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="iSBNLabel_4">
              <property name="text">
               <string>URL</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="nameLineEdit_4"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="commentLabel_3">
              <property name="text">
               <string>Comment</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="commentLineEdit_3"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Tags</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="lineEdit_2"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="add_button">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListView" name="listView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>450</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>iSBNLineEdit</tabstop>
  <tabstop>titleLineEdit</tabstop>
  <tabstop>linkLineEdit</tabstop>
  <tabstop>commentLineEdit_2</tabstop>
  <tabstop>add_button</tabstop>
  <tabstop>typeComboBox</tabstop>
  <tabstop>listView</tabstop>
  <tabstop>nameLineEdit_2</tabstop>
  <tabstop>iSBNLineEdit_2</tabstop>
  <tabstop>titleLineEdit_2</tabstop>
  <tabstop>tagsLineEdit_2</tabstop>
  <tabstop>commentLineEdit_4</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>iSBNLineEdit_3</tabstop>
  <tabstop>titleLineEdit_3</tabstop>
  <tabstop>tagsLineEdit</tabstop>
  <tabstop>commentLineEdit</tabstop>
  <tabstop>iSBNLineEdit_4</tabstop>
  <tabstop>titleLineEdit_4</tabstop>
  <tabstop>nameLineEdit_4</tabstop>
  <tabstop>commentLineEdit_3</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>typeComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>input_form</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>88</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
